File: frontend/src/constants/OrganizationType.ts
import { Dayjs } from "dayjs";

export type OrganizationType = {
  
  id: number;
  descriptionKg: string;
  textColor: string;
  backgroundColor: string;
  name: string;
  description: string;
  code: string;
  nameKg: string;
};


export type OrganizationTypeCreateModel = {
  
  id: number;
  descriptionKg: string;
  textColor: string;
  backgroundColor: string;
  name: string;
  description: string;
  code: string;
  nameKg: string;
};

File: backend/Domain/Entities/OrganizationType.cs
namespace Domain.Entities
{
    public class OrganizationType : BaseLogDomain
    {
        public int Id { get; set; }
        public string? Name { get; set; }
        public string? Description { get; set; }
        public string? Code { get; set; }
        public string? NameKg { get; set; }
        public string? DescriptionKg { get; set; }
        public string? TextColor { get; set; }
        public string? BackgroundColor { get; set; }
    }
}
